resources/views/welcome.blade.php

```diff
@@ -32,7 +32,7 @@
         <div class="flex flex-col justify-around h-full">
             <div>
                 <h1 class="text-gray-600 text-center font-light tracking-wider text-5xl mb-6">
-                    {{ config('app.name', 'Laravel') }}
+                    Bootcamp4
```

Comment:
In `.env` and .`.env.example` you would specific `APP_NAME` as "Bootcamp 4", rather hard code the app name. 

Just revert this change and make it read `config('app.name')`, committing the APP_NAME to `.env.example`.